app/adzerk/api.py

```diff
@@ -30,23 +30,29 @@ async def get_decisions(self):
         """
         timeout = aiohttp.ClientTimeout(total=30)
         connector = aiohttp.TCPConnector(limit=None)
-        async with aiohttp.ClientSession(timeout=timeout, connector=connector) as session:
-            async with session.post(conf.adzerk['decision']['url'], json=self.get_decision_body()) as r:
-                if r.status == 400:
-                    text = await r.text()
-                    # This occurs when there is no site with the requested id from adzerk.
-                    # So instead we send back no results but log an error
-                    logging.error(text)
-                    return dict()
-                response = await r.json()
+        # Wrap the request in a try/catch block to log timeout errors.
+        try:
+            async with aiohttp.ClientSession(timeout=timeout, connector=connector) as session:
+                async with session.post(conf.adzerk['decision']['url'], json=self.get_decision_body()) as r:
+                    if r.status == 400:
+                        text = await r.text()
+                        # This occurs when there is no site with the requested id from adzerk.
+                        # So instead we send back no results but log an error
+                        logging.error(text)
+                        return dict()
+                    response = await r.json()
 
-        decisions = response['decisions']
-        if not decisions or len(decisions) == 0:
+            decisions = response['decisions']
+            if not decisions or len(decisions) == 0:
+                return dict()
+            for _, dec in decisions.items():
+                if dec:
+                    map(validation.validate_decision, dec)
+            return decisions
+        except Exception as e:
+            # Log the exact parameters sent to Kevel
+            logging.error(repr(self.get_decision_body()))
```

Comment:
Thanks @mmiermans! It would have the POST parameters we send to Kevel, so we'd be able to give the vendor examples of requests that timed out. I'll ask the team WRT re-enabling Sentry, as far as I know it was taken out because it wasn't actually being used for anything.